app/src/pages/common/settingsPage/autoAnalysisTab/analysisForm/strategyBlock/strategyBlock.jsx

```diff
@@ -173,6 +228,41 @@ export class StrategyBlock extends Component {
             </span>
           </InputRadio>
         </FormField>
+
+        <FormField
+          fieldWrapperClassName={cx('switcher-wrapper')}
+          label={formatMessage(messages.uniqueErrorAutoAnalysisSwitcherTitle)}
+          customBlock={{
+            node: <p>{Parser(formatMessage(messages.uniqueErrorAutoAnalysisStatusInfo))}</p>,
+          }}
+          withoutProvider
+        >
+          <InputBigSwitcher
+            value={data[UNIQUE_ERROR_ENABLED]}
+            onChange={this.changeUniqueErrorAutoAnalysisEnabled}
+            disabled={uniqueErrorAutoAnalysisDisabled}
```

Comment:
Looks like it always `false` it should be depended on analyzer service

Reply:
agree